### DISCUSSION TITLE
Create functions within now-build not working

### DISCUSSION THREAD

**Discussion by Alex0007** [vercel/vercel#4108] [General] [answered]
`api` folder with functions is created during `npm run now-build` execution
But no functions are found after that. 

Is this intended behaviour? 

Node environment, framework preset: other

**Comment by Alex0007** [comment]
If i place empty file at `api/handler.js`, and then rewrite it with `npm run now-build` task, rewrited version will be used. 

So problem is: service doesn't detect new functions in `api` folder that appear after build task

**Comment by coetry** [comment, marked as answer]
Hi @Alex0007 this is expected behavior the `now-build` script is designed to dynamically generated files that your functions can consume, not to dynamically build functions. 

You will need to populate your `api` directory with all of your functions ahead of time.
